INSTALL.md

```diff
@@ -97,6 +102,28 @@ rpm.sh - create a RPM package
 slack.sh - create a package for Slackware
 ```
 
+#### Linux flatpak
+First make sure you have `flatpak` and `flatpak-builder` installed. Follow the
+instructions for your distribution on [flathub](https://flatpak.org/setup/). The
+instructions there only cover the installation of `flatpak`, but 
+`flatpak-builder` should be the same.
+
+Once you have `flatpak` and `flatpak-builder` installed:
+```
+git clone git://github.com/Bitmessage/PyBitmessage.git
+cd PyBitmessage/
+git submodule add https://github.com/flathub/shared-modules.git
+flatpak-builder --install _flatpak -install-deps-from=flathub --force-clean --user org.bitmessage.PyBitmessage.json
```

Comment:
It's probably a typo here, should be `--install-deps-from=`

Reply:
> It's probably a typo here, should be `--install-deps-from=`

Fixed. The updated build instructions are now in [packages/flatpak/README.md](https://github.com/Bitmessage/PyBitmessage/pull/1618/files#diff-8421532ac371e215f58a7bbb8a782c8a).